### PR TITLE
For #1277: Add RepeatedProc

### PR DESCRIPTION
**Pull request by andreoss**
For #1277 
Introduced  `RepeatedProc`

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#1399](https://github.com/yegor256/cactoos/issues/1399) is assigned to @marceloamadeu[/z](https://www.0crat.com/u/marceloamadeu), here is [why](https://www.0crat.com/footprint/C63314D6Z/a03f6863-f013-4171-b226-d778e8194896); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by marceloamadeu** [comment]
@paulodamaso looks good to me

**Comment by marceloamadeu** [comment]
@andreoss Please correct:

`ERROR: Puzzle src/main/java/org/cactoos/RepeatedProc.java:35-35 has a very short description of just 6 words while a minimum of 20 is required`

**Comment by andreoss** [comment]
> @andreoss Please correct:
> 
> `ERROR: Puzzle src/main/java/org/cactoos/RepeatedProc.java:35-35 has a very short description of just 6 words while a minimum of 20 is required`

@marceloamadeu Done

**Comment by marceloamadeu** [comment]
@paulodamaso ping

File: src/main/java/org/cactoos/RepeatedProc.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ * SOFTWARE.
+ */
+package org.cactoos;
+
+import org.cactoos.func.FuncOf;
+import org.cactoos.func.Repeated;
+
+/**
+ * Proc that repeats its calculation a few times before
+ * returning the last result.
```
Comment (victornoel) [inline src/main/java/org/cactoos/RepeatedProc.java:31]: @andreoss a Proc does not return any result, can you fix this javadoc?

Comment (andreoss) [reply src/main/java/org/cactoos/RepeatedProc.java]: @victornoel Done

File: src/test/java/org/cactoos/io/XsltTransformer.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.stream.StreamResult;
+import javax.xml.transform.stream.StreamSource;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+
+public class XsltTransformer {
```
Comment (victornoel) [inline src/test/java/org/cactoos/io/XsltTransformer.java:15]: @andreoss you incorrectly added this class apparently, can you remove it?

**Comment by marceloamadeu** [comment]
@andreoss Please... check Travis CI build failure.
Tks

File: src/test/java/org/cactoos/io/XsltTransformer.java (hunk 2 above)
Comment (andreoss) [reply src/test/java/org/cactoos/io/XsltTransformer.java]: @victornoel Fixed

**Comment by victornoel** [comment]
@rultor merge

**Comment by 0crat** [comment]
Code review was too long (18 days), architects (@paulodamaso, @victornoel[/z](https://www.0crat.com/u/victornoel)) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Pull request merged**